### DISCUSSION TITLE
Docker: Change container user away from root

### DISCUSSION THREAD

**Discussion by yroffin** [pglombardo/PasswordPusher#560] [Ideas]
hi,

is possible to switch image user to another on than root for security issue ?

i've checked on latest and runtime user is set to root :(

**Comment by pglombardo** [comment]
Hi @yroffin - I'll see if I can do this sometime soon.

**Reply by yroffin** [reply]
your project pwpush is very cool ...
i 'm using it ... and that's rock !!!! thanks for your spent time on it

**Reply by pglombardo** [reply]
Hi @yroffin - I've merged this change.  The docker container processes now run under `1000:1000` uid:gid.

[v1.24.5](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.24.5) is building now.  To be sure, I'll test to see if it breaks anything on the next deploy and post back here soon.
